src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -2204,6 +2270,28 @@ internal ScriptRequirements GetScriptRequirements()
             }
         }
 
+        private List<string> HandleRequiresOSVersionsArgument(Ast argumentAst, object arg, ref List<string> requiredOSVersions)
+        {
+            if (!(arg is string))
```

Comment:
Rather than allow any input, I think this method should just take a string and make the caller check its inputs